src/psyclone/tests/psyclone_test_utils.py

```diff
@@ -307,6 +307,11 @@ def _code_compiles(self, psy_ast):
             # Build the kernels. We allow kernels to also be located in
             # the temporary directory that we have been passed.
             for fort_file in kernel_modules:
+
+                # Skip file if it is not Fortran
+                if fort_file.endswith(".cl"):
```

Comment:
Please could you refer to #372 here as this is something we want to test in future.

Reply:
Done